.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
is this file needed?

Reply:
it's only for debug in visual code, it is already eliminated